content/en/container-orchestration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container Orchestration
+status: Draft
+category: Concept
+---
+
+## What it is
+[Container](/container/) orchestration provides the ability to manage and automate the lifecycle of containerized applications in dynamic environments. 
```

Comment:
```suggestion
[Container](/container/) orchestration refers to managing and automating containerized applications' lifecycles in dynamic environments. 
```

Provided the ability to manage = too many words

Reply:
We should probably add in the first section that this is infrastructure specific.